### DISCUSSION TITLE
Self-hosting license options

### DISCUSSION THREAD

**Discussion by bmaginni** [renovatebot/renovate#23127] [Request Help] [answered]
### How are you running Renovate?

Placeholder value, please select the correct response from the dropdown

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

We wanted to use the self-hosting version of renovate, but the AGPL license is unfortunately not allowed to be used within our company.
Is there any supported/paid for service or self-hosting service for renovate that will work with bitbucket server?

### What you tried so far.

N/A

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by secustor** [comment]
I don't see why you shouldn't be able to use Renovate self hosted as long you are not integrating it into your product. 

Tough there is a Mend supported variant https://www.mend.io/free-developer-tools/renovate/on-premises/

**Reply by rarkins** [reply]
The on-prem unfortunately doesn't support Bitbucket Server due to lack of native webhooks

**Reply by secustor** [reply]
TIL :/

**Comment by rarkins** [comment, marked as answer]
You can gain access to Renovate functionality without AGPL through this commercial product: https://www.mend.io/repo-integration/

**Reply by bmaginni** [reply]
Taking a quick read of this, is this triggered by detection of vulnerabilities or does it also provide the renovate dependency management for newer versions etc as well?
We already have separate tooling for detection of vulnerabilities, it was the pro-active dependency maintenance of renovate that was appealing to us
How does this renovate behaviour work with bitbucket server, given lack of native webhooks issue?

**Reply by rarkins** [reply]
That solution includes proactive dependency updating also. The Mend team wrote a custom Bitbucket Server plug-in which substitutes for webhooks by sending its own events to the repo integrations
